package.json

```diff
@@ -151,6 +149,8 @@
     "libp2p-delegated-peer-routing": "^0.10.0",
     "libp2p-floodsub": "^0.27.0",
     "libp2p-gossipsub": "^0.10.0",
+    "libp2p-interfaces": "^1.0.0",
```

Comment:
interfaces should be a dependency, but the compliance totally a devDep

Reply:
done

Reply:
It could also make sense to turn `aegir` into devDependency within `libp2p-interfaces-compliance-tests` but haven't checked that